app/boards/arm/adv360pro/adv360pro.dtsi

```diff
@@ -0,0 +1,163 @@
+/*
+*
+* Copyright (c) 2021 Polarity Works
+* SPDX-License-Identifier: MIT
+*
+*/
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+#include <dt-bindings/led/led.h>
+#include <dt-bindings/zmk/matrix_transform.h>
+
+#include "adv360pro_pinctrl.dtsi"
```

Comment:
Small nitpick, but the normal Zephyr convention for these files seems to be hyphenated:

```suggestion
#include "adv360pro-pinctrl.dtsi"
```